{'project_id': '3130', 'year': '1984', 'x': 588330, 'y': 2389860, 'plot_id': '23036', 'strata': '41'}
Pandas(Index=0, project_id='3130', plot_id='23036', x=588330, y=2389860)
called process_plot
Pandas(Index=0, project_id='3130', plot_id='23036', x=588330, y=2389860)
23036
Directory '/efs/timesync/1984/audit/' already exists.
Data has been pickled and saved to '/efs/timesync/1984/audit/1984_23036_3130.p'.
..............Failed qa: 1984-04-22T15:55:41.1910380Z
Failed qa: 1984-04-29T16:01:37.475047Z
.......Failed qa: 1984-05-08T15:55:54.4970810Z
.......Failed qa: 1984-07-09T16:09:27.8440500Z
Failed qa: 1984-07-11T15:56:55.117531Z
.......Failed qa: 1984-08-28T15:58:21.441550Z
..............Failed qa: 1984-09-13T15:58:25.339544Z
Failed qa: 1984-09-29T15:58:14.0570810Z
.......Failed qa: 1984-10-15T15:58:11.7890310Z
..............Failed qa: 1984-10-31T15:58:17.957044Z
.......Failed qa: 1984-11-23T16:04:23.128519Z
Data has been pickled and saved to '/efs/timesync/1984/audit/1984_23036_3130_qa_rejects.p'.
Success tracking goes here
